lib/logger.js

```diff
@@ -129,23 +129,41 @@ Logger.prototype.child = function(args) {
 
 Logger.prototype.log = function(level, info) {
     var levelMatch = this._levelMatcher.exec(level);
-    if (levelMatch) {
+    if (info && levelMatch) {
         var logger = this._logger;
         var simpleLevel = levelMatch[1];
-        if (info && info instanceof String) {
-            info = {msg: info};
```

Comment:
This was a double-bug. `"test" instanceof String` is actually `false`, but we didn't need to wrap a message into an object - it gets overriden by bunyan with a third argument value (or by undefined).